### PR TITLE
Remove article caption from render, update help text

### LINKED ISSUE #3461: Article Featured Image caption breaking page layout
# Problem
When you add a caption to the Featured Image in an article, the caption wrapper is visibly hidden but has a width of 100% and is breaking the page.

Screen Shot 2021-05-05 at 1 29 30 PM

Screen Shot 2021-05-05 at 1 27 09 PM

# Steps to reproduce
Create an article and add a featured image, then add a caption to the image. Add the Articles component to a page.

Examples:
https://stanleymuseum.prod.drupal.uiowa.edu/
https://stanleymuseum.prod.drupal.uiowa.edu/news

# Proposed solution
I'm not sure what the intended use of the image caption is for an article. As far as I can tell, it is not visible anywhere on the site.

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/3461 and fixes regression related to the hero to banner refactor of article featured image.

# How to test

- `yarn workspace uids_base gulp --development`
- Go to https://stanleymuseum.local..drupal.uiowa.edu/news and verify that page overflow scrollbar is gone.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
Created a follow-up issue to decide how to deal with article caption going forward: https://github.com/uiowa/uiowa/issues/3561

File: docroot/themes/custom/uids_base/scss/components/caption.scss [hunk 1]
```diff
@@ -1,6 +1,13 @@
 @import "assets/scss/_variables.scss";
 @import "assets/scss/_utilities.scss";
 
+.banner:hover {
```
Comment (bspeare) [inline docroot/themes/custom/uids_base/scss/components/caption.scss]: Can we comment this out until we have done further work on this?

Comment (joewhitsitt) [reply docroot/themes/custom/uids_base/scss/components/caption.scss]: can do!

**Comment by bspeare** [comment]
I have tested this and I can verify that the image caption no longer appears on the teaser list and breaking the page for https://stanleymuseum.local.drupal.uiowa.edu/news and that there is help text under the article featured image caption on the node edit screen explaining that the caption is currently not rendering.  The caption also remains visually hidden on article, pages, and banner components until we figure out a better way to render the caption.  I would approve this if I hadn't created the original pull request.

**Review by cory-skeers — APPROVED** [review]
Checked changes in addition to @bspeare's approval. Looks good to go 👍

**Pull request merged**